core-bundle/src/Controller/InsertTagsController.php

```diff
@@ -25,13 +28,26 @@
  */
 class InsertTagsController
 {
-    public function __construct(private readonly InsertTagParser $insertTagParser)
-    {
+    public function __construct(
+        private readonly InsertTagParser $insertTagParser,
+        private readonly ResponseTagger|null $responseTagger,
+    ) {
     }
 
     public function renderAction(Request $request, string $insertTag): Response
     {
-        $response = new Response($this->insertTagParser->replaceInline($insertTag));
+        if (!str_starts_with($insertTag, '{{') || !str_ends_with($insertTag, '}}')) {
+            throw new \InvalidArgumentException(sprintf('Invalid insert tag "%s"', $insertTag));
```

Comment:
That should be a `BadRequestHttpException` imho - we're in HTTP context here.

Reply:
Changed in e76e984d763e0e159e65fc6319b1ac8e4967ee09